.studio/tooling

```diff
@@ -152,36 +152,6 @@ function install_gomock() {
   install_go_tool "${go_mock_tools[@]}"
 }
 
-document "install_go_tool" <<DOC
```

Comment:
Why is this deleted? Should we keep this one and remove the one in 01_base.sh?

Reply:
Mostly for load order so that we know when we can start depending on it.